Updated aws-lc-fips-sys and fips feature messaging (#282)
aws/aws-lc-rs: aws-lc-rs/README.md

## Feature Flags

##### alloc (default)
Allows implementation to allocate values of arbitrary size. (The meaning of this feature differs
from the "alloc" feature of *ring*.) Currently, this is required by the `io::writer` module.

##### ring-io (default)
Enable feature to access the  `io`  module.

##### ring-sig-verify (default)
Enable feature to preserve compatibility with ring's `signature::VerificationAlgorithm::verify`
function. This adds a requirement on `untrusted = "0.7.1"`.

##### fips
Enable this feature to have aws-lc-rs use the [*aws-lc-fips-sys*](https://crates.io/crates/aws-lc-fips-sys)
crate for the cryptographic implementations. The *aws-lc-fips-sys* crate provides bindings to
[AWS-LC-FIPS 2.x](https://github.com/aws/aws-lc/tree/fips-2022-11-02), which has completed
FIPS validation testing by an accredited lab and has been submitted to NIST for certification.

> The static build of AWS-LC-FIPS will be used.

Reviewer: ```suggestion
The static build of AWS-LC-FIPS is used.
```

Reviewer: suggestion for throughout
